bin/katello-certs-check

```diff
@@ -182,6 +182,18 @@ function check-ca-bundle-size () {
     fi
 }
 
+function check-ca-bundle-trust-rules () {
+    printf "Checking if CA bundle has trust rules: "
+    CHECK=$(awk -v cmd='openssl x509 -noout -text' ' /BEGIN/{close(cmd)};{print | cmd}'  $CA_BUNDLE_FILE |grep 'Trusted Uses'|wc -l)
+    printf $CHECK
+    if [[ $CHECK -lt 1 ]]; then
+		success
```

Comment:
```suggestion
	success
```

Reply:
I finally understood what you mean about the indentation. I was copying the indentation of the function check-ca-bundle-size which was wrong.  I was confused because my indentation was matching that one which I assumed was a good template to follow. Guess I was wrong.  I'll adjust the indentation of that one too in my next update.